pages/hackathon/[slug]/index.js

```diff
@@ -233,7 +233,8 @@ export default function Hackathon() {
 
                                                             <div
                                                                 style={{
-                                                                    height: "1rem",
+                                                                    height:
+                                                                        "1rem",
```

Comment:
Changes in this file should not be a part of this PR, Can you please remove these changes

Reply:
In the instruction for contributing it is mentioned to run "npm run format", so after running that command this also gets formatted automatically, should i reverse this change?

Reply:
No its fine